components/search/SearchErrorMessage.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import React from "react";
+
+const ErrorMessage = styled.p`
+  padding: 50px 0;
+  text-align: center;
+`
+
+export const SearchErrorMessage: React.FC<{ searchText: string }> = ({searchText}) => {
+  return (
+    <>
```

Comment:
is it necessary?

Reply:
yes without it will be error

Reply:
I mean
export const SearchErrorMessage: React.FC<{ searchText: string }> = ({searchText}) => searchText && (
        <ErrorMessage>
might work same way
if it will complain that react component should return string or null you can do it this way:
export const SearchErrorMessage: React.FC<{ searchText: string }> = ({searchText}) => searchText ? (
        <ErrorMessage>
        ...
  ) : null